iocore/eventsystem/I_IOBuffer.h

```diff
@@ -1254,7 +1254,7 @@ class MIOBuffer
   clear()
   {
     dealloc();
-    size_index = BUFFER_SIZE_NOT_ALLOCATED;
+    size_index = default_large_iobuffer_size;
```

Comment:
Should this be master's pr?

Reply:
Yes, we can test how it works on quic-latest, but it should be landed on master before we merge quic-latest.